src/Cli/func/Helpers/PythonHelpers.cs

```diff
@@ -19,6 +27,49 @@ public static class PythonHelpers
 
         public static string VirtualEnvironmentPath => Environment.GetEnvironmentVariable("VIRTUAL_ENV");
 
+        /// <summary>
+        /// Determines which Python package tool to use based on the files present in the directory and tool availability.
+        /// Priority: uv (if pyproject.toml + uv.lock and uv is available) > poetry (if pyproject.toml and poetry is available) > pip (if requirements.txt)
```

Comment:
NIT: Add a `.` at the end 
```suggestion
        /// Priority: uv (if pyproject.toml + uv.lock and uv is available) > poetry (if pyproject.toml and poetry is available) > pip (if requirements.txt).
```